app/src/main/java/org/pragyan/dalal18/fragment/CompanyDescriptionFragment.kt

```diff
@@ -17,7 +17,7 @@ class CompanyDescriptionFragment : Fragment() {
 
     private lateinit var model: DalalViewModel
 
-    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View {
```

Comment:
any specific reason to make the view **non-nullable** here ?
refer to the official [view-binding](https://developer.android.com/topic/libraries/view-binding) docs please !